src/test/java/com/uber/cadence/workflow/WorkflowMigrationTest.java

```diff
@@ -41,10 +43,7 @@
 import java.util.UUID;
 import java.util.concurrent.CancellationException;
 import org.apache.thrift.TException;
-import org.junit.After;
-import org.junit.Assume;
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.*;
 
 public class WorkflowMigrationTest {
```

Comment:
I do want to have an integration test from end to end.

Reply:
Sorry, I don't think I explained this well, but the CadenceTestRule has the conditional logic within it already: https://github.com/uber/cadence-java-client/blob/704380bfaed0aa873fa19fe01d7bb0645c690c53/src/test/java/com/uber/cadence/testUtils/CadenceTestRule.java#L196 .

If `USE_DOCKER_SERVICE` is set to true then CadenceTestRule will connect to the docker instance instead of creating a test service. The goal of the test rule is to avoid every test needing to have conditional logic.

Reply:
Oh, I don't know that. I'll remove it completely